docs/source/glance.rst

```diff
@@ -0,0 +1,405 @@
+               self.l1 = L.Linear(n_units)  # n_in -> n_units
+               self.l2 = L.Linear(n_units)  # n_units -> n_units
+               self.l3 = L.Linear(n_out)  # n_units -> n_out
+   
+       def __call__(self, x):
+           h1 = F.relu(self.l1(x))
+           h2 = F.relu(self.l2(h1))
+           return self.l3(h2)
+   
+Since mushrooms are either edible or poisonous (no information on psychedelic effects!) in the dataset, we'll use a Link :class:`~chainer.links.Classifier` for the output, with 44 units (double the features of the data) in the hidden layers and a single true/false category for classification.
+   
+.. code-block:: python
+
+     model = L.Classifier(
+         MLP(44, 1), lossfun=F.sigmoid_cross_entropy, accfun=F.binary_accuracy)
+
+Optimizer
+~~~~~~~~~~~~
+.. image:: ../image/glance/trainer-optimizer.png
+
+Pick an :class:`~chainer.optimizer`, and set up the :class:`~chainer.model` to use it.
+
+.. code-block:: python
+
+   # Setup an optimizer
+   optimizer = chainer.optimizers.SGD()
+   optimizer.setup(model)
+   
+Updater
+~~~~~~~~~
+.. image:: ../image/glance/trainer-updater.png
+
+Now that we have the training :class:`~chainer.iterator` and :class:`~chainer.optimizer` set up, we link them both together into the :class:`~chainer.updater`. The :class:`~chainer.updater` uses the minibatches from the :class:`~chainer.iterator`, and then does the forward and backward processing of the model, and updates the parameters of the model according to the :class:`~chainer.optimizer`. Setting the ``device=-1`` sets the device as the CPU. To use a GPU, set ``device`` equal to the number of the GPU, usually ``device=0``.
+
+.. code-block:: python
+
+   # Create the updater, using the optimizer
+   updater = training.StandardUpdater(train_iter, optimizer, device=-1)
+   
+Set up the :class:`~chainer.updater` to be called after the training batches and set the number of batches per epoch to 100. The learning rate per epoch will be output to the directory ``result``.
+
+.. code-block:: python
+
+   # Set up a trainer
+   trainer = training.Trainer(updater, (50, 'epoch'), out='result')
+   
+Extensions
+~~~~~~~~~~
+.. image:: ../image/glance/trainer-extensions.png
+
+Use the testing :class:`~chainer.iterator` defined above for an :class:`~chainer.training.extensions.Evaluator` extension to the trainer to provide test scores.
+
+If using a GPU instead of the CPU, set ``device`` to the ID of the GPU, usually ``0``.
+
+.. code-block:: python
+
+   trainer.extend(extensions.Evaluator(test_iter, model, device=-1))
+   
+Save a computational graph from ``loss`` variable at the first iteration. ``main`` refers to the target link of the ``main`` :class:`~chainer.optimizer`. The graph is saved in the `Graphviz's <http://www.graphviz.org/>`_ dot format. The output location (directory) to save the graph is set by the :attr:`~chainer.training.Trainer.out` argument of :class:`~chainer.training.Trainer`.
```

Comment:
This paragraph is rendered in bold font (incorrectly, I guess). Please check the results.

Reply:
Fixed, I believe.